docs/README.md

```diff
@@ -55,13 +55,15 @@ graph
 
 ## Details
 
-Clownface provides a set of chainable methods. The most important ones are `.in(predicate)` and `.out(predicate)` which allow the traversal through the graph. It is possible to chain as many of these methods to extract a sub-graph from the available dataset.
+At the core of clownface sits an object called a "graph pointer". A graph pointer can be backed by zero or more RDF/JS terms. It is being created by calling the function exported by the packages's default module.
```

Comment:
It may be worth explaining what we mean by "graph pointer"? Or si it self-explanatory?